setup.py

```diff
@@ -32,6 +27,21 @@
     raise SystemError("Only Windows, Linux, or MacOS is supported!")
 
 
+# custom class for ctypes
+class CTypesExtension(Extension):
+    pass
+
+
+class bdist_wheel_abi_none(bdist_wheel):
+    def finalize_options(self):
+        bdist_wheel.finalize_options(self)
+        self.root_is_pure = False
+
+    def get_tag(self):
+        python, abi, plat = bdist_wheel.get_tag(self)
```

Comment:
What is `plat`? And what is `abi`?
If we're not using `python` and `abi`, please prefix (or replace) with underscores.